src/internal/components/token-list/token-focus-controller.tsx

```diff
@@ -8,34 +8,27 @@ import styles from './styles.css.js';
 const tokenSelector = `.${styles['list-item']}`;
 const toggleButtonSelector = `.${styles.toggle}`;
 
-export function useTokenFocusController({ removedItemIndex }: { removedItemIndex?: null | number }) {
+export function useTokenFocusController({ moveFocusNextToIndex }: { moveFocusNextToIndex?: null | number }) {
   const tokenListRef = useRef<HTMLDivElement>(null);
 
   useEffect(() => {
-    if (removedItemIndex === undefined || removedItemIndex === null || tokenListRef.current === null) {
+    if (moveFocusNextToIndex === undefined || moveFocusNextToIndex === null || tokenListRef.current === null) {
       return;
     }
 
     const tokenElements = tokenListRef.current.querySelectorAll(tokenSelector);
     const toggleButton = tokenListRef.current.querySelector(toggleButtonSelector);
 
-    const activeItemIndices: number[] = [];
-    for (let i = 0; i < tokenElements.length; i++) {
-      if (!tokenElements[i].querySelector('[aria-disabled="true"]')) {
-        activeItemIndices.push(i);
-      }
-    }
```

Comment:
With this change, disabled token's should not be skipped when finding the next focusable token.